mediawiki/LocalSettings.d/LocalSettings.override.php

```diff
@@ -23,7 +23,7 @@
 /*******************************/
 /* Disable UI error-reporting  */
 /*******************************/
-#ini_set( 'display_errors', 1 );
+ini_set( 'display_errors', '0' );
```

Comment:
this also deactivates errors,- for an intermediate solution its fine with this to hide deprecation warnings, but i think we should still display error to see issues in production immediately. 

Locally this change hides the deprecation-warnings.